wiki/en/en-Installation-for-Macintosh.md

```diff
@@ -10,14 +10,15 @@ permalink: "/wiki/Installation-for-Macintosh"
 Make sure you've already read the [Getting Started](Getting-Started) page.
 
 1. [Download Jamulus](https://sourceforge.net/projects/llcon/files/latest/download){: target="_blank" rel="noopener noreferrer" .button}
-1. **Open the downloaded `.dmg` file**. Agree to the license. You will now see two icons (Jamulus client and server).
-1. **Drag and drop both icons into your Applications folder** to install Jamulus.
+1. **Install Jamulus**. Open the downloaded `.dmg` file, agree to the license, *drag and drop* the two icons you see in the window (Jamulus client and server) into your *Applications folder*. After that, you can close this window.
```

Comment:
Do we really need so many steps?

Reply:
Perhaps not. I don't know whether having an "install" step inside a page called "install" is a bit unclear, since it implies the other steps aren't about installing, but I doubt most people will think about that.

Reply:
Let's use my proposal and since it might add benefit for inexperienced users.